src/Common/Core/Twig/Extensions/BaseTwigModifiers.php

```diff
@@ -37,7 +37,24 @@ public static function formatCurrency($string, $currency = 'EUR', $decimals = nu
             default:
         }
 
-        return $currency.'&nbsp;'.number_format((float) $string, $decimals, ',', '&nbsp;');
+        return $currency.'&nbsp;'.static::formatNumber($string, $decimals);
```

Comment:
`php-cs-fixer --level=psr2` won't say a thing about this too ;) It doesn't care since PSR-2 doesn't say a thing about this.